transport/src/main/java/io/scalecube/transport/TransportSettings.java

```diff
@@ -57,14 +54,31 @@ public boolean isUseNetworkEmulator() {
     return useNetworkEmulator;
   }
 
```

Comment:
Better to remove ``getWriteBufferHighWaterMark()`` ; otherwise it looks no logical -- we do have HWM and don't have low water mark.